server/csvSeed.js

```diff
@@ -0,0 +1,98 @@
+const fs = require('fs');
+const fastcsv = require('fast-csv');
+const { Pool } = require('pg');
+const faker = require('faker');
+const path = require('path');
+
+let imageCounter = 1;
+
+const getRandomImages = () => {
+  let results = '';
+  const limit = faker.random.number({ min: 4, max: 7 });
+
+  for (let j = 0; j < limit; j += 1) {
+    const imgURL = `https://aragorn-images.s3-us-west-2.amazonaws.com/${imageCounter}.jpg`;
+    results += `NEXT${imgURL}`;
+    imageCounter += 1;
+    if (imageCounter > 30) {
+      imageCounter = 1;
+    }
+  }
+  return results;
+};
+
+let currentRow = 0;
+const idCounter = 0;
+
+function seedPostgres() {
+  const stream = fs.createReadStream(path.join(__dirname, '/../games.csv'));
+  const csvData = [];
+  const csvStream = fastcsv
+    .parse()
+    .on('data', (data) => {
+      csvData.push(data);
+    })
+    .on('end', () => {
+      // remove the first line: header
+      csvData.shift();
+
+      let counter = 0;
+      function loop() {
+        const pool = new Pool({
+          user: 'postgres',
+          host: 'localhost',
+          database: 'games22',
+          password: 'samaung1',
+          port: 5432,
+        });

```

Comment:
Rather than recreating this here and having two places to update if your DB info changes, I'd recommend making a separate db config file and just require it into any files that need it.

Reply:
good point, will do